bmm/src/main/java/org/openehr/bmm/v2/persistence/PBmmGenericProperty.java

```diff
@@ -1,5 +1,29 @@
 package org.openehr.bmm.v2.persistence;
 
+import org.openehr.bmm.core.BmmClass;
+import org.openehr.bmm.core.BmmGenericType;
+import org.openehr.bmm.core.BmmUnitaryProperty;
+import org.openehr.bmm.v2.validation.converters.BmmClassProcessor;
+
 public final class PBmmGenericProperty extends PBmmProperty<PBmmGenericType> {
 
+    public PBmmGenericProperty() {
+        super();
+    }
```

Comment:
This default constructor will automatically be added because there are no other constructors defined. It can therefore be removed.